workflow-dev/tpl/deis-monitor-grafana-rc.yaml

```diff
@@ -35,6 +35,10 @@ spec:
           value: http://$(DEIS_MONITOR_INFLUXAPI_SERVICE_HOST):$(DEIS_MONITOR_INFLUXAPI_SERVICE_PORT_TRANSPORT)
         - name: "BIND_PORT"
           value: "3500"
+        - name: "GRAFANA_USER"
```

Comment:
i think you should rename it to `DEFAULT_USER`  and ` DEFAULT_USER_PASSWORD ` https://github.com/deis/monitor/blob/master/grafana/rootfs/usr/share/grafana/start-grafana#L7 or change the grafana code also